HOME: /t3home/frejalom
USER: frejalom
SLURM_JOB_ID: 321993
HOSTNAME: t3wn62
CMSSW RELEASE BASE  /cvmfs/cms.cern.ch/slc7_amd64_gcc12/cms/cmssw/CMSSW_14_2_0_pre4
%MSG-i AlpakaService:  (NoModuleName) 11-Apr-2025 10:18:51 CEST pre-events
AlpakaServiceSerialSync succesfully initialised.
Found 1 device:
  - Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz
%MSG
11-Apr-2025 10:18:57 CEST  Initiating request to open file root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root
%MSG-w XrdAdaptorInternal:  file_open 11-Apr-2025 10:20:04 CEST pre-events
Failed to open file at URL root://cms-xrd-global.cern.ch:1094//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root?tried=+1213xrootd-redic.pi.infn.it&xrdcl.requuid=e3edf937-9039-4900-b7cb-9b97860c8d88.
%MSG
%MSG-w XrdAdaptorInternal:  file_open 11-Apr-2025 10:20:04 CEST pre-events
Failed to open file at URL root://cms-xrd-global.cern.ch:1094//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root?tried=+1213xrootd-redic.pi.infn.it,&xrdcl.requuid=8c03c577-f2af-49d9-94ce-c7a71cd5f176.
%MSG
----- Begin Fatal Exception 11-Apr-2025 10:20:04 CEST-----------------------
An exception of category 'FileOpenError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing input source of type PoolSource
   [2] Calling RootInputFileSequence::initTheFile()
   [3] Calling StorageFactory::open()
   [4] Calling XrdFile::open()
Exception Message:
Failed to open the file 'root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root'
   Additional Info:
      [a] Input file root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root could not be opened.
      [b] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.
' (errno=3011, code=400). No additional data servers were found.
      [c] Last URL tried: root://cms-xrd-global.cern.ch:1094//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root?tried=+1213xrootd-redic.pi.infn.it,&xrdcl.requuid=8c03c577-f2af-49d9-94ce-c7a71cd5f176
      [d] Problematic data server: cms-xrd-global.cern.ch:1094
      [e] Disabled source: cms-xrd-global.cern.ch:1094
----- End Fatal Exception -------------------------------------------------
logfile
pv_cfg.py
cp: cannot stat 'pv.root': No such file or directory
